docs/usage/configuration-options.md

```diff
@@ -1904,6 +1904,15 @@ If enabled Renovate will pin Docker images by means of their SHA256 digest and n
 
 ## platformAutomerge
 
+<!-- prettier-ignore -->
+!!! warning
+    Before you enable `platformAutomerge` you must tell your Git hosting platform what test(s) must pass before the PR is allowed to be auto-merged.
+    If you don't do this, Renovate auto-merges PRs even if all test fail!
+    On GitHub this is called "Require status checks before merging", which you can find in the "Branch protection rules" section of the settings for your repository.
+    [GitHub docs, about protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches)
+    [GitHub docs, require status checks before merging](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging)
+    If you're using another platform, search their documentation for a similar feature.
```

Comment:
I don't know how to format this text so:

- Prettier ignores the formatting
- Material for MkDocs renders the admonition correctly on the published page

Here's what the end-result should look like:

> Before you enable `platformAutomerge` you must tell your Git hosting platform what test(s) must pass before the PR is allowed to be auto-merged.
> If you don't do this, Renovate auto-merges PRs even if all test fail!
> 
> On GitHub this is called "Require status checks before merging", which you can find in the "Branch protection rules" section of the settings for your repository.
> - [GitHub docs, about protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches)
> - [GitHub docs, require status checks before merging](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging)
>
> If you're using another platform, search their documentation for a similar feature.

If you have any tips/tricks/suggestions, please let me know. 😉